frontend/resourceadm/components/AccessListMembers/AccessListMembers.test.tsx

```diff
@@ -0,0 +1,260 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { render as rtlRender, screen, waitFor } from '@testing-library/react';
```

Comment:
Don't know if you have seen it on the ```Team Studio``` slack channel today, but it was suggested to always import it as render, and then if we needed to create our own render method, we could use more meaningful names. 
Suggestions so far is to either name it ```studioRender```, or to a name that informs about what the render does, e.g., ```renderAccessListMembers```. 😄